staging/kos/pkg/apis/network/v1alpha1/types.go

```diff
@@ -289,12 +289,17 @@ type NetworkAttachmentStatus struct {
 	// +optional
 	Errors NetworkAttachmentErrors `json:"errors,omitempty" protobuf:"bytes,1,opt,name=errors"`
 
+	// AddressContention indicates whether the address assignment was
+	// delayed due to not enough addresses being available at first.
+	AddressContention bool `json:"addressDelayed,omitempty" protbuf:"bytes,9,opt,name=addressDelayed"`
```

Comment:
In the marshalling tag the Protobuf index is out of order.

Reply:
Yes, we are forced to violate one ordering or another.  I chose to prioritize the textual order over the protobuf field index order.